apps/crn-server/test/data-providers/research-outputs.data-provider.test.ts

```diff
@@ -832,6 +832,7 @@ describe('ResearchOutputs data provider', () => {
 
   describe('Create and update', () => {
     afterEach(() => {
+      // eslint-disable-next-line jest/no-standalone-expect
```

Comment:
is this really needed?

Reply:
Lint was failing there. And adding it to that error that wasn't picked up by the linter I'd say it's not being ran correctly everywhere, will also gather more info about it.